.travis.yml

```diff
@@ -6,8 +6,8 @@ git:
 
 language: python
 
-python:
-  - 3.6
+python:  
```

Comment:
Looks like extra trailing whitespace could be removed.

Reply:
Good catch, thanks!